### ISSUE TITLE
Channel not found error

### ISSUE THREAD

**Issue by jcbwlkr** [michielrensen/hubot-slack-reaction#2]
Hello! Thanks for making this!

For some reason I can't get it to work for my slack team. When I use the module it just fails silently but with a little bit of debugging I see the response from Slack is

```json
{"ok":false,"error":"channel_not_found"}
```

I got it to work by making this change

```diff
-    payload.channel  = data.message.room
+    payload.channel  = data.message.rawMessage.channel
```

I was just going to submit a PR changing that line but I saw in the history that in commit e8947b4ee307ea4d769205016b81859bda8c7218 @andypoorman had specifically changed it from `rawMessage.channel` to `room`. I was wondering if there was some reason for the change that I'm missing or perhaps there are different versions of Slack teams or something?

**Comment by andypoorman** [comment]
Yeah the other way didn't work for me, that's why I changed it. Interesting...

The way data is passed to the API is still the same, but the only difference we might have is how we're getting the data from slack.

I'm using this:
"hubot-slack": "^4.2.1"

**Comment by jcbwlkr** [comment]
Oh, yep that's the problem. I was on an hubot-slack version 3.something. I updated to current and it worked fine. Thanks @andypoorman 

I'm not sure exactly what version of `hubot-slack` changed this but maybe it would be worth the time to figured that out and put a dependency in this package. @michielrensen feel free to close if you don't want a reminder sticking around :)

**Comment by ls-michielrensen** [comment]
Hey guys. Great work in figuring it out and thanks for the suggestion of adding a version dependency. I will fix this tomorrow and hopefully figure which version broke it.

**Comment by ls-michielrensen** [comment]
@jcbwlkr @andypoorman I released a new version v0.0.4 which reverted the PR AND adds `hubot-slack@v4.2.1` as dependency. In any case, I noticed this addition to the documentation: https://slackapi.github.io/hubot-slack/basic_usage#message-reactions which basically obsoletes my package...
